services/spar/src/Spar/SCIM.hs

```diff
@@ -456,12 +456,14 @@ createScimToken zusr CreateScimToken{..} = do
             -- it makes sense semantically
             token <- ScimToken . cs . ES.encode <$> liftIO (randBytes 32)
             tokenid <- randomId
+            now <- liftIO getCurrentTime
```

Comment:
The `Spar` monad has `getNow` from saml2-web-sso, but I'm not sure you want to use that in this module.  There is something nice about not having SCIM depend on SAML.

So, feel free to ignore this comment.